requirements.json

```diff
@@ -177,5 +177,30 @@
         "sha256sum": "d65a1d1a050e4fd7912c18468954f64db824a1e1b621235a153d010beae14757"
       }
     }
+  },
+  "kompose": {
+    "name": "Kompose",
+    "description": "A tool to translate a Docker Compose file into Kubernetes or OpenShift resources",
+    "vendor": "Kubernetes Incubator",
+    "modulePath": "model/kompose",
+    "targetFolderName": "kompose",
+    "version": "0.5.0",
```

Comment:
Todd requested two things to fix:
1. It should not be included into bundled version (which is done already)
2. Mark it as 'Technology Preview'  https://github.com/sudhirverma/developer-platform-install/pull/3